### ISSUE TITLE
Typo in README.md

### ISSUE THREAD

**Issue by rderimay** [balzss/luxbar#16]
In the HTML example, it should be:
```
<input type="checkbox" id="luxbar-checkbox" class="luxbar-checkbox">
```

in place of

```
<input type="checkbox" id="luxbar-checkbox">
```

**Comment by balzss** [comment]
The input tag only requires the id attribute because it doesn't have any styling. So there's no need to change that.

**Comment by rderimay** [comment]
Yes, but it does need the **class** attribute also which is missing in the readme example.
Otherwise, line 10 of the dist css:
```
  .luxbar-static .luxbar-checkbox:checked ~ .luxbar-menu {
    position: absolute; }
```

won't work.
And in fact it does not work.

**Comment by balzss** [comment]
That's strange. I made sure before replying to check it in the demo site,  and it does't have it  neither. But to be 100% I will reopen the issue and double check it when I get home. :)

**Comment by rderimay** [comment]
Ok this is true. On the demo site the the lumbar.css does contain #luxbar-checkbox in place of the .luxbar-checkbox in the dist code...

**Comment by rderimay** [comment]
With class selector also in the sass code btw.

**Comment by balzss** [comment]
Oh I see now. That's because the changes between version 0.1.3 and 0.2.0. I will correct it when I'll get home. And thanks for pointing this out.

**Comment by rderimay** [comment]
You're welcome. Thanks to you for putting up this project :-)

**Comment by balzss** [comment]
Fixed in 3031f9d44b26503b2e2cd71d0433072bc5528fc9.
